external/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -208,3 +208,5 @@ object SchemaConverters {
 
 private[avro] class IncompatibleSchemaException(
   msg: String, ex: Throwable = null) extends Exception(msg, ex)
+
+private[avro] class InvalidAvroTypeException(msg: String) extends Exception(msg)
```

Comment:
It seems that we can reuse `IncompatibleSchemaException`, or renaming it as `UnsupportedAvroTypeException`